### DISCUSSION TITLE
Is the Authenticator Extension Available in China?

### DISCUSSION THREAD

**Discussion by willjacksonironfish** [Authenticator-Extension/Authenticator#894] [Q&A] [answered]
Hi I need some help.

I am trying to roll out a MFA for our China offices.

I would like to use the Authenticator Extension but I am unsure if this is possible in China?

Does anyone know if there is a work around so that my Users in China and use the Authenticator Extension?

Any help would be greatly appreciated.

Thanks!

**Comment by Sneezry** [comment, marked as answer]
Google's Chrome Web Store is blocked in China's network. You need a proxy to install the extension.

**Reply by willjacksonironfish** [reply]
Thanks so much!
